core/templates/services/literally-canvas-helper.service.spec.ts

```diff
@@ -260,24 +269,27 @@ describe('LiterallyCanvasHelperService', function() {
   });
 
   it('should fail svg validation', function() {
-    var validSvgTag = '<svg width="100" height="100"><rect id="rectangle-de5' +
-    '69866-9c11-b553-f5b7-4194e2380d9f" x="143" y="97" width="12" height="29' +
-    '" stroke="hsla(0, 0%, 0%, 1)" fill="hsla(0, 0%, 100%, 1)" stroke-width=' +
-    '"1"></rect></svg>';
+    var validSvgTag = (
+      '<svg width="100" height="100"><rect id="rectangle-de569866-9c11-b553-' +
+      'f5b7-4194e2380d9f" x="143" y="97" width="12" height="29" stroke="hsla' +
+      '(0, 0%, 0%, 1)" fill="hsla(0, 0%, 100%, 1)" stroke-width="1"></rect>' +
+      '</svg>');
     expect(LiterallyCanvasHelperService.isSVGTagValid(validSvgTag)).toBe(true);
 
-    var invalidSvgAttribute = '<svg widht="100" height="100"><rect id="recta' +
-    'ngle-de569866-9c11-b553-f5b7-4194e2380d9f" x="143" y="97" width="12" he' +
-    'ight="29" stroke="hsla(0, 0%, 0%, 1)" fill="hsla(0, 0%, 100%, 1)" strok' +
-    'e-width="1"></rect></svg>';
+    var invalidSvgAttribute = (
```

Comment:
Is the intended error here "widht"? If so, maybe change it to 'misspelled-width' or, better, 'invalid-arg-name' to make it more clear.

Reply:
Done.